src/Sarif.Sarifer/SariferPackage.cs

```diff
@@ -82,6 +85,8 @@ await this.GetServiceAsync(typeof(SVsShell)).ConfigureAwait(continueOnCapturedCo
             }
 
             SolutionEvents.OnBeforeCloseSolution += this.SolutionEvents_OnBeforeCloseSolution;
+
+            await SariferOption.InitializeAsync(this);
```

Comment:
>; [](start = 53, length = 1)

should we use configureawait?

Reply:
done

---
In reply to: [564970665](https://github.com/microsoft/sarif-visualstudio-extension/pull/354#discussion_r564970665) [](ancestors = 564970665)